circleci/utils

```diff
@@ -21,6 +21,29 @@ install_awscli(){
   echo "Completed AWS cli install"
 }
 
+assume_role_with_web_identity() {
+  echo "Setting aws profile to $2"
+  aws configure set profile $2
+  echo "Authenticating via role $1..."
```

Comment:
Good for PoC, but unsure if we want these values within CI logs.

```suggestion
  aws configure set profile $2
```

Reply:
Yeah i can remove the role string from the logs!